Source/Relativity.Test.Helpers/Relativity.Test.Helpers/UserHelpers/CreateUser.cs

```diff
@@ -233,18 +290,43 @@ public static int FindGroupArtifactId(IRSAPIClient proxy, string group)
 					{
 						Condition = new TextCondition(GroupFieldNames.Name, TextConditionEnum.EqualTo, @group)
 					};
-
 				queryGroup.Fields.Add(new FieldValue(ArtifactQueryFieldNames.ArtifactID));
-
 				QueryResultSet<kCura.Relativity.Client.DTOs.Group> resultSetGroup = proxy.Repositories.Group.Query(queryGroup, 0);
 
-				if (resultSetGroup.Success && resultSetGroup.Results.Count == 1)
+				// Check for success.
+				if (resultSetGroup == null)

```

Comment:
you may want to move this to a function instead of repeating the same code

Reply:
Done